.github/workflows/python-sdk-update.yml

```
@@ -0,0 +1,37 @@
+name: Update Python SDK Documentation
+
+on:
+  repository_dispatch:
+    types: [python-sdk-updated]
+
+jobs:
+  update-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout docs repository
+        uses: actions/checkout@v3
+        
+      - name: Checkout firecrawl repository
+        uses: actions/checkout@v3
```

```suggestion
        uses: actions/checkout@v5
        
      - name: Checkout firecrawl repository
        uses: actions/checkout@v5
```
https://github.com/actions/checkout?tab=readme-ov-file#checkout-v5